letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
data_dir is defined in odoo config file, not in database, AFAIK. It's safe to assume that we can write in that directory because there Odoo writes filestore and sessions.

> we don't want different databases to overwrite each other's certificates/keys

I'm looking for a way to change well-known URI to add database parameter in ACME protocol. This way we can load registry for that DB in controller, and safe certificates/keys in a subdirectory by database name.